Packs/EmailCommunication/Scripts/SendEmailReply/SendEmailReply.py

```diff
@@ -239,25 +900,45 @@ def main():
     email_subject = custom_fields.get('emailsubject')
     email_cc = custom_fields.get('emailcc', '')
     add_cc = custom_fields.get('addcctoemail', '')
+    add_bcc = custom_fields.get('addbcctoemail', '')
     service_mail = args.get('service_mail', '')
     email_from = custom_fields.get('emailfrom')
     email_to = custom_fields.get('emailto')
     email_latest_message = custom_fields.get('emaillatestmessage')
-    email_code = custom_fields.get('emailgeneratedcode')
+    email_code = custom_fields.get('emailgeneratedcode')  # single code field for 'Email Communication' types
+    email_codes = custom_fields.get('emailgeneratedcodes')  # multi-code field for other incident types
     email_to_str = get_email_recipients(email_to, email_from, service_mail, mailbox)
     files = args.get('files', {})
     attachments = argToList(args.get('attachment', []))
+    new_email_attachments = custom_fields.get('emailnewattachment', {})
     notes = demisto.executeCommand("getEntries", {'filter': {'categories': ['notes']}})
-
-    try:
-        final_email_cc = get_email_cc(email_cc, add_cc)
-        reply_body, reply_html_body = get_reply_body(notes, incident_id, attachments)
-        entry_id_list = get_entry_id_list(incident_id, attachments, files)
-        result = validate_email_sent(incident_id, email_subject, email_to_str, reply_body, service_mail, final_email_cc,
-                                     reply_html_body, entry_id_list, email_latest_message, email_code)
-        demisto.results(result)
-    except Exception as error:
-        return_error(str(error), error)
+    mail_sender_instance = args.get('mail_sender_instance', None)
+    new_thread = args.get('new_thread')
+    new_email_recipients = custom_fields.get('emailnewrecipients')
+    new_email_subject = custom_fields.get('emailnewsubject')
+    new_email_body = custom_fields.get('emailnewbody')
+    email_selected_thread = custom_fields.get('emailselectedthread')
+
+    if new_email_attachments:
+        new_attachment_names = ', '.join([attachment.get('name', '') for attachment in new_email_attachments])
+    else:
+        new_attachment_names = 'None'
```

Comment:
aquestion for later. why are we using here 'None' and not None?
the same as fir the string 'false' instead of False.

Reply:
The `new_attachment_names` variable is a human-readable string that is eventually included in the new `EmailThreads` context entry to store that email.  When the email thread is displayed later in the thread viewer this string is shown so the user can see what file names were present on that particular email message.